web/client/components/geostory/layouts/sections/__tests__/Banner-test.jsx

```diff
@@ -0,0 +1,212 @@
+        setTimeout(done);
+    });
+    it('Banner rendering with defaults', () => {
+        ReactDOM.render(<Banner />, document.getElementById("container"));
+        const container = document.getElementById('container');
+        const el = container.querySelector('.ms-section-banner');
+        expect(el).toExist();
+    });
+    it('Banner background rendering (image)', () => {
+        const IMAGE_SRC = 'data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAEAAAABCAYAAAAfFcSJAAAADUlEQVR42mP8z8BQDwAEhQGAhKmMIQAAAABJRU5ErkJggg==';
+        const CONTENTS = [
+            {
+                id: '000',
+                type: 'column',
+                background: {
+                    type: 'image',
+                    src: IMAGE_SRC,
+                    lazy: false
+                },
+                html: '<h1>Banner</h1>'
+            }
+        ];
+        ReactDOM.render(<Banner contents={CONTENTS} mode="edit"/>, document.getElementById("container"));
+        const container = document.getElementById('container');
+        const el = container.querySelector('.ms-section-banner');
+        expect(el).toExist();
+        const img = el.querySelector('img');
+        expect(img).toExist();
+        expect(img.getAttribute('src')).toBe(IMAGE_SRC);
+        const contentToolbar = container.querySelector('.ms-content-toolbar');
+        expect(contentToolbar).toExist();
+
+        const buttonsInToolbar = container.querySelectorAll('.ms-section-background-container .btn-group .glyphicon');
+        expect(buttonsInToolbar).toExist();
+        expect(buttonsInToolbar.length).toBe(7);
+        testToolbarButtons(["pencil", "height-view", "fit-contain", "size-extra-large", "align-center", "dropper", "trash"], container);
+
+    });
+    it('Banner rendering cover set to true', () => {
+        const VIEW_HEIGHT = 500;
+        const CONTENTS = [
+            {
+                id: '000',
+                type: 'text',
+                html: '<h1>Banner</h1>'
+            }
+        ];
+        ReactDOM.render(<Banner contents={CONTENTS} viewHeight={VIEW_HEIGHT} sectionType="banner" cover mode="edit"/>, document.getElementById("container"));
+        const container = document.getElementById('container');
+        const el = container.querySelector('.ms-section-banner');
+        expect(el).toExist();
+
+        const backgroundContainer = container.querySelector('.ms-section-background-container');
+        expect(backgroundContainer).toExist();
+        expect(backgroundContainer.clientHeight).toBe(VIEW_HEIGHT);
+        const contentToolbar = container.querySelector('.ms-content-toolbar');
+        expect(contentToolbar).toExist();
+        testToolbarButtons(["pencil", "height-auto", "trash"], container);
+    });
+
+    it('Banner rendering cover set to false', () => {
+        const VIEW_HEIGHT = 500;
+        const CONTENTS = [
+            {
+                id: '000',
+                type: 'text',
+                html: '<h1>Banner</h1>'
```

Comment:
Please remove `html: '<h1>Banner</h1>'` and `type: text` because they are not used by banner